blog/content/post/2023-09-08-msi_z690-a_v1.1.2_release.md

```diff
@@ -0,0 +1,283 @@
+firmware community.
+
+{{< youtube iTkXunUAriE >}}
+
+Of course, that means some microcontroller sitting on the mainboard can read
+the filesystem and flash BIOS SPI under certain circumstances and have access
+to a specific USB port. That microcontroller does not run open-source firmware
+yet and could be leveraged by attackers with the ability to update it or have
+physical access. The only reasonable protection for Intel platforms is Intel
+Boot Guard with owner keys fused into the chipset, which we deliver as part of
+our Dasharo Enterprise offering.
+
+### Raptor Lake-S CPU Support
+
+We've started to add support for the upcoming Raptor Lake-S CPUs in Dasharo.
+This update, highlighted in our [GitHub issue #130](https://github.com/Dasharo/dasharo-issues/issues/130),
+is a straightforward response to the tech community's growing interest in the
+latest CPUs. It's a simple step to help Dasharo users upgrade smoothly when
+they decide so. Thanks to this feature, we stay at the bleeding edge of devices
+supported by coreboot.
+
+### MSI ACPI Device and Automatic Driver and Utility Installation Manager
+
+The integration of the MSI ACPI device in Dasharo triggers the automatic driver
+and utility installation manager, a feature that promises to simplify the setup
+process for users. As demonstrated in the following video, this tool guides
+users step-by-step to effortlessly install and set up the necessary drivers and
+utilities, removing the hassle of manual installations. It's a practical
+addition to Dasharo, aimed at making the initial setup process smoother and
+more user-friendly, especially for those taking their first steps in the
+open-source firmware environment. It also proves that Dasharo and open-source
+firmware is not only for Linux users but can also be leveraged by Windows users
+quickly and smoothly without compatibility issues.
+
+{{< youtube K-v-veV_jvI >}}
+
+### Support for Logo Customization
+
+Dasharo now offers a feature that many users will find delightful - the ability
+to customize the logo displayed during the boot process. As outlined in our
+[guide](https://docs.dasharo.com/guides/logo-customization/), users can now
+replace the existing logo with a design of their choice, adding a personal
+touch to their system. Whether it's a fun image, a brand logo, or a personal
+design, this feature allows users to infuse their personality into their
+firmware, making the boot process a bit more enjoyable and unique to each
+individual.
+
+### UEFI 2.8 Errata C Compliance in EDKII Fork
+
+![UEFI Logo](/img/368px-Logo_of_the_UEFI_Forum.svg.png)
+
+In the v1.1.2 release, we have updated Dasharo to be compliant with the UEFI
+2.8 Errata C specification, released in January 2021, a standard that dictates
+how the firmware communicates with the operating system.
+
+This transition brings a notable enhancement to the firmware's cryptographic
+system, introducing support for the SHA-3 hash algorithm, known for
```

Comment:
I don't see SHA-3 anywhere in the following:
https://github.com/Dasharo/edk2/pull/62
https://github.com/Dasharo/edk2/pull/63
https://github.com/Dasharo/edk2/pull/64
https://github.com/Dasharo/edk2/pull/65

Where does it come from?

Reply:
@miczyg1 you are right that was plain wrong, I did analysis of the MRs you pointed and completely rewrote this section: https://github.com/3mdeb/news-and-ideas/pull/537/commits/87b20686469991eaa52f7b41b1680bea00288344